web/src/components/CircularGauge.tsx

```diff
@@ -11,67 +33,81 @@ export interface CircularGaugeProps {
+// Memoized to avoid unnecessary re-renders due to floating point precision issues
+const SpringAnimatedArc = memo(function SpringAnimatedArc({
+  radius,
+  ratio,
+}: {
+  radius: number;
+  ratio: number;
+}) {
+  const [spring] = useSpring(
+    {
+      to: { ratio: Number.isFinite(ratio) ? ratio : 0 },
+      from: { ratio: 0 },
+    },
+    [ratio]
+  );
+
+  return (
+    <AnimatedArc
+      innerRadius={radius * 0.8}
+      outerRadius={radius}
+      startAngle={PIE_START_ANGLE}
+      endAngle={spring.ratio.to((ratio) => calculateEndAngle(ratio))}
+      cornerRadius={radius}
+      className="fill-brand-green dark:fill-brand-green-dark"
+    />
+  );
+});
+
 export function CircularGauge({
   ratio,
   name,
-  tooltipContent,
   testId,
+  tooltipContent,
 }: CircularGaugeProps) {
-  // TODO: To improve performance, the background pie does not need to rerender on percentage change
-  const data = [{ value: ratio }];
-  const percentageAsAngle = ratio * 360;
-  const endAngle = PIE_START_ANGLE - percentageAsAngle;
+  const height = 80;
+  const width = 80;
+  const radius = Math.min(width, height) / 2;
+  const centerY = height / 2;
+  const centerX = width / 2;
 
   return (
     <div className="flex flex-col items-center">
-      <TooltipWrapper
-        side="right"
-        tooltipContent={tooltipContent}
-        tooltipClassName="max-w-44"
-      >
-        {/* Div required to ensure Tooltip is rendered in right place */}
-        <div data-test-id={testId}>
-          <PieChart
-            width={65}
-            height={65}
-            margin={{ top: 0, left: 0, right: 0, bottom: 0 }}
-          >
-            <Pie
-              innerRadius="80%"
-              outerRadius="100%"
-              startAngle={90}
-              endAngle={-360}
-              paddingAngle={0}
-              dataKey="value"
-              data={[{ value: 100 }]}
-              className="fill-gray-200/60 dark:fill-gray-600/50"
-              isAnimationActive={false}
-              strokeWidth={0}
-            >
-              <Label
-                className="select-none fill-gray-900 text-[1rem] font-bold dark:fill-gray-300"
-                position="center"
-                offset={0}
-                formatter={(value: number) =>
-                  Number.isNaN(value) ? '?%' : `${Math.round(value * 100)}%`
-                }
-                value={ratio}
-              />
-            </Pie>
-            <Pie
-              data={data}
-              innerRadius="80%"
-              outerRadius="100%"
-              startAngle={90}
-              endAngle={endAngle}
-              fill="#3C764A"
-              paddingAngle={0}
-              dataKey="value"
-              animationDuration={500}
-              animationBegin={0}
-              strokeWidth={0}
-            />
-          </PieChart>
+      {/* Div required to ensure Tooltip is rendered in right place */}
+      <TooltipWrapper tooltipContent={tooltipContent} side="bottom">
```

Comment:
Haha it originally was like that so you'll have to talk with @Alportan about this 😉

The current way does match what we do today though.

Reply:
The main reason behind the whole widget being interactive is the mobile version, where we want to make it as easy to interact with the tooltip. 🙋

Reply:
Sorry beforehand, but perhaps we do it differently on mobile and web then? 😅 😬

Reply:
I'll work something out 🫡

Reply:
I'm good with that approach, my main goal here is to make a pleasant mobile experience ✨📱✨

Reply:
How important is it to include this change in this PR?

Turns out it's not as straight forward as I thought it would be and it would be nice to get this included in the next mobile release.

If we merge it without the change I'll add a slack week item to look into then instead.

Reply:
Works on my side, as long as we provide a good mobile experience to begin with 🙌  (but then again I'm biased towards mobile users 🙃 )

Reply:
It's easier to fix desktop later due to the more frequent releases as well.

Reply:
Fine with me, as long as we don't completely forget it once it goes into the backlog? 😬

Reply:
Feel free to keep bugging me about it so that don't happen 😉

Reply:
I can create a task so we have it to the next slack week 💪

Reply:
I have already created a linear task and assigned myself 🙂